### ISSUE TITLE
Add search box to the docs landing page

### ISSUE THREAD

**Issue by mesellings** [camunda/camunda-docs#4061]
We should investigate whether it is technically feasible to add a large search box to the main page area of the docs landing page, and if this is something that users want or need.

For example:

<img width="1007" alt="Screenshot 2024-07-24 at 16 21 28" src="https://github.com/user-attachments/assets/8d957c40-cef2-4314-97e9-1868c79294a5">

**Comment by christinaausley** [comment]
Yes! Agreed -- will this be related to https://github.com/camunda/developer-experience/issues/340, or do you consider this out of scope?

**Comment by mesellings** [comment]
I think this is probably out of scope for this iteration, as it may be technically challenging and Steve hasn't looked at this before (I've rarely seen this on Docusaurus sites tbh, I'm guessing there is a reason).

**Comment by mesellings** [comment]
@pepopowitz FYI

**Comment by mesellings** [comment]
Very quick 10 minutes realization/play that this is easily possible! See https://github.com/camunda/camunda-docs/tree/landing-page-search
